src/ImageSharp/Processing/Processors/Transforms/Resize/ResizeKernel.cs

```diff
@@ -28,15 +28,23 @@ internal ResizeKernel(int startIndex, float* bufferPtr, int length)
         /// <summary>
         /// Gets the start index for the destination row.
         /// </summary>
-        public int StartIndex { get; }
+        public int StartIndex
+        {
+            [MethodImpl(InliningOptions.ShortMethod)]
```

Comment:
I think the JIT is smart enough to do this without our intervention. I see the .NET BCL code pushing back a lot on applications for explicit `AggressiveInlining` these days, I suppose the JIT does it more reliably with really short methods.

Reply:
It's more about expressing intent than anything. The JIT can ignore our suggestion but I believe we should still declare that we want the method inlined.

Reply:
Hmm that makes sense, but in trivial cases like property getters it gets overly verbose and hurts readability IMO. (Technically *all* autoproperies should be marked then.)